### PR TITLE
Create 2016-01-22-home-away-from-home.md

### PR DESCRIPTION
**Pull request by atreni**

### CODE REVIEW

File: source/posts/2016-01-22-home-away-from-home.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+A few weeks later, I moved to Brooklyn for summer 2015. I saw an opportunity to bridge two worlds that were very important to me - DockYard’s consulting services and civic technology. I asked for permission to take a leave of absence to attend a fellowship at [Blue Ridge Labs](http://labs.robinhood.org/), a social impact incubator focused on problem solving with technology for low income communities in New York. It was an amazing thing to receive their blessing and support.
+
+DockYard is an amazing place, that values people and relationships. But I think the most incredible part, which I’ve been so fortunate enough to receive time and again, is the grace of its individual’s willingness to mentor and teach. While some companies are in search of unicorns and are in the game of stealing resources, DockYard seeks out something special in its employees more important than skill, and fosters the growth of individual people as well as talent.
```
Comment (poteto) [inline source/posts/2016-01-22-home-away-from-home.md]: `its individual’s` -> `its individuals'`

File: source/posts/2016-01-22-home-away-from-home.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+author: Ashley Treni
+twitter: "ashleytreni"
+github: atreni
+published: true
+tags: community, culture, team
+---
+
+For those who don’t know my journey with DockYard, it’s quite an amazing story. 
+
+As an imposingly curious grad student studying data visualization, I absolutely couldn’t resist the urge to jump into a conversation I overheard while bartending at Highball Lounge in March of 2014 (who can resist buzzwords like information architecture, javascript, and data!). A nice chat and a short introduction later, I was invited by Brian and Dan of DockYard to reach out and learn more about the company. I was immediately intrigued - both by the conversation and also the type of people who didn’t dismiss me because of my job (as most tend to do to those who work in the service industry).
```
Comment (poteto) [inline source/posts/2016-01-22-home-away-from-home.md]: `javascript` -> `JavaScript`

**Comment by Ctannerweb** [comment]
Can you add a summary. When we start the newsletter all blog posts will need a summary in them. Just something short and simple :)

`summary: "..."`

File: source/posts/2016-01-22-home-away-from-home.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+I’ll never forget the first conversation I had with Steve when I went to the DockYard office to learn about the design team. I was absolutely amazed that they would consider me for an internship, and saw value in my curiosity and engagement, rather than the checklist of my industry experience (which, for the record, was non existent). After a summer internship in 2014 where I was welcomed with open arms and [jumped right into the conversation](https://dockyard.com/blog/2014/07/18/design-as-conversation), I joined DockYard full time in January 2015. This was during my final semester of grad school, and I received a tremendous amount of support from my colleagues while I worked simultaneously on my [Master’s thesis project](http://www.designtaxonomy.com/#/standardimprovisations/).
```
Comment (poteto) [inline source/posts/2016-01-22-home-away-from-home.md]: `non existent` -> `non-existent`

**Comment by poteto** [comment]
We're going to miss you! 😭

File: source/posts/2016-01-22-home-away-from-home.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+A few weeks later, I moved to Brooklyn for summer 2015. I saw an opportunity to bridge two worlds that were very important to me - DockYard’s consulting services and civic technology. I asked for permission to take a leave of absence to attend a fellowship at [Blue Ridge Labs](http://labs.robinhood.org/), a social impact incubator focused on problem solving with technology for low income communities in New York. It was an amazing thing to receive their blessing and support.
+
+DockYard is an amazing place, that values people and relationships. But I think the most incredible part, which I’ve been so fortunate enough to receive time and again, is the grace of its individuals' willingness to mentor and teach. While some companies are in search of unicorns and are in the game of stealing resources, DockYard seeks out something special in its employees more important than skill, and fosters the growth of individual people as well as talent.
```
Comment (rsocci) [inline source/posts/2016-01-22-home-away-from-home.md]: no comma needed after `place`

File: source/posts/2016-01-22-home-away-from-home.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+twitter: "ashleytreni"
+github: atreni
+summary: "A short reflection"
+published: true
+tags: community, culture, team
+---
+
+For those who don’t know my journey with DockYard, it’s quite an amazing story. 
+
+As an imposingly curious grad student studying data visualization, I absolutely couldn’t resist the urge to jump into a conversation I overheard while bartending at Highball Lounge in March of 2014 (who can resist buzzwords like information architecture, JavaScript, and data!). A nice chat and a short introduction later, I was invited by Brian and Dan of DockYard to reach out and learn more about the company. I was immediately intrigued - both by the conversation and also the type of people who didn’t dismiss me because of my job (as most tend to do to those who work in the service industry).
```
Comment (michaeldupuisjr) [inline source/posts/2016-01-22-home-away-from-home.md]: I'd make the words quotations:

buzzwords like information "architecture", "JavaScript", and "data"!)

**Comment by michaeldupuisjr** [comment]
Nice post @atreni!

**Pull request merged**